config/webpack/loaders/devSiteExample.js

```diff
@@ -1,34 +1,65 @@
 const path = require('path');
 const startCase = require('lodash.startcase');
+const findCss = require('./devSiteCssFinder');
 
 /**
  * Generate the example with the supplied file.
  * Don't use an arrow function
  */
-const loader = async function loader() {
+const loader = async function loader(content) {
   const callback = this.async();
 
   const exampleSource = this.resourcePath;
   const parsedResourcePath = path.parse(exampleSource);
+  const cssFileName = findCss(content);
 
   const code = [
     'import React from \'react\';',
     `import Example from '${exampleSource}';`,
     `import Code from '${exampleSource}?dev-site-codeblock';`,
     'import ExampleTemplate from \'terra-dev-site/lib/loader-components/_ExampleTemplate\';',
-    '',
-    `export default ({ title, description, isExpanded }) => (
+  ];
+
+  if (cssFileName !== undefined) {
+    try {
+      this.resolve('', cssFileName, async () => {
+      });
```

Comment:
should we be using the string result of `this.resolve` within the callback for the import?

Reply:
I assume we're just using `this.resolve` to verify the file exists, we just need the file path and it will be resolved again in the code block loader